### PR TITLE
feat: Custom Cypress tests

### PR DESCRIPTION
**Pull request by lucasfp13**
## What's the purpose of this pull request?

This PR intends to setup the Cypress test extension.

### Faststore related PRs

https://github.com/vtex/faststore/pull/2080

### CODE REVIEW

File: faststore.config.js [hunk 1]
```diff
@@ -70,6 +70,7 @@ module.exports = {
   experimental: {
     nodeVersion: 18,
     cypressVersion: 12,
+    enableCypressExtension: true,
```
Comment (hellofanny) [inline faststore.config.js:73]: Are we going to enable it as default? I think we shouldn't, the user should enable it when needed, WTDY?

Comment (lucasfp13) [reply faststore.config.js:73]: Yep, we won't change that, it was just for testing purposes 🤝

**Review by emersonlaurentino — APPROVED** [review]

**Comment by hellofanny** [comment]
@lucasfp13 should we merge this PR or https://github.com/vtex-sites/starter.store/pull/244?

**Comment by lucasfp13** [comment]
> @lucasfp13 should we merge this PR or #244?

No, we don't need to merge this PR as it was created for testing purposes only.

**Comment by lucasfp13** [comment]
> @lucasfp13 should we merge this PR or #244?

No, we don't need to merge this PR as it was created for testing purposes only.

**Pull request closed without merging**